hibike/README.md

```diff
@@ -46,75 +104,118 @@ Each Smart Device will be assigned an 88-bit UID with the following data.
     |      (16 bits)    |   (8 bits)   |          (64 bits)          |
     +-------------------+--------------+-----------------------------+
 
-Device Type - 16-bit ID specifying the type device the Smart Device controller is attached to.
-              Device types are enumerated in Section 4
+- Device Type
+    - 16-bit ID specifying the Type of a Smart Device
+    - Device types are enumerated in Section 5
+
+- Year
+  - 8-bit ID corresponding to the competition year that the Smart Device was manufactured for. 
+  - The 2015-2016 season will correspond to 0x00
```

Comment:
The spec should be updated with a new year, `0x01`.

Additionally, is possible to detect last year's hibike devices and warn about protocol incompatibility to students? I would not be at all surprised if students try to plug in last year's sensors to their robots. Breaking compatibility is fine as long this does not become a yearly thing, but having old sensors crash the robot would be unfortunate.